infrastructure/data_access/http/synthetics_repo.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime, timedelta, timezone
+from typing import List
+from pprint import pprint
+
+from generated.synthetics_http_client.synthetics import ApiClient
+from generated.synthetics_http_client.synthetics import Configuration
```

Comment:
There are multiple unused imports in Python files. We need tool to find them and remove them eventually.

Reply:
I cleaned imports with isort in https://github.com/kentik/sla_dashboard_webapp/pull/2. Also I will configure CI with isort job in another PR.